devops/openshift/sysdig-team.yml

```diff
@@ -0,0 +1,43 @@
+apiVersion: ops.gov.bc.ca/v1alpha1
+kind: SysdigTeam
+metadata:
+  name: 0c27fb-sysdigteam
+  namespace: 0c27fb-tools
+spec:
+  team:
+    description: The Sysdig Team for the OpenShift Project Set SIMS
+    users:
+    - name: jason.tang@gov.bc.ca
+      role: ROLE_TEAM_EDIT
+    - name: kal.marsh@gov.bc.ca
+      role: ROLE_TEAM_EDIT
+    - name: mohammed.alnahari@gov.bc.ca
+      role: ROLE_TEAM_EDIT
+    - name: john.marchesan@gov.bc.ca
+      role: ROLE_TEAM_EDIT
+    - name: harold.a.johnson@gov.bc.ca
+      role: ROLE_TEAM_EDIT
+    - name: gurumoorthy.mohan@aot-technologies.com
+      role: ROLE_TEAM_EDIT
+    - name: dheepak.rr@aot-technologies.com
```

Comment:
shouldn't we use our gov email here?

Reply:
The login in controlled via github single sign on, the sysdig application verifies the email id configured in your github id. As we are using our github id created from aot, we have to use our ato technologies email.

Reply:
For my clarification.  We're not sending alerts to everyone on this list right? ALl these people should have access, but not receive email alerts directly.  We will want to send the alerts to a generic email address.

Reply:
Regarding the alerts, there are none right now, we can create a Email notification channel where all the emails can go to a generic email address.